doc/features.md

```diff
@@ -0,0 +1,74 @@
+Network Module Features
+-----------------------
+
+Network Services
+======================
+
+- wicked
+- Network Manager
+- no network service
+
+IPv6
+==============
+
+- enable/disable it
+
+DHCP Client options
+====================
+
+- set hostname according to dhcp response
+- set machine identifier for asking dhcp
+- allow to set specific option for hostname to send for dhcpd client ( how often it is used? )
+
+DNS
+===================
+
+- set hostname
+- set domain
```

Comment:
this is bad. hostname and domain is actually combined into FQDN and written into /etc/hostname. This feature is a bit controversial. See https://trello.com/c/so0amSNp/1176-sle-12-sp3-fate-yast2-network-clarify-hostname-setup